### PR TITLE
Documented set_point_cloud() for ConvexPolygonShape2D

### PR DESCRIPTION
**Pull request by ghost**
Just a minor addition to the documentation.  Noticed this method is using a convex hull on the points you give it, and seems to be so at least up to this point in the source:

https://github.com/godotengine/godot/blob/5bd0ff003fdd82cf84a557251633ebb83bad4fd5/scene/resources/convex_polygon_shape_2d.cpp#L48-L53

### CODE REVIEW

File: doc/classes/ConvexPolygonShape2D.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
 			<argument index="0" name="point_cloud" type="PoolVector2Array">
 			</argument>
 			<description>
-				Currently, this method does nothing.
+				Based on the set of points provided, this creates and assigns the [member points] using the convex hull algorithm. Removing all unneeded points. (Convex hull is also available at [method Geometry.convex_hull_2d])
```
Comment (akien-mga) [inline doc/classes/ConvexPolygonShape2D.xml]: Looks good, I would just slightly rephrase the last part (changes in bold):

> Based on the set of points provided, this creates and assigns the [member points] **property** using the convex hull algorithm**, r**emoving all unneeded points. See [method Geometry.convex_hull_2d] for details.

Comment (avencherus) [reply doc/classes/ConvexPolygonShape2D.xml]: Changes made.  :thumbsup:

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.0.7.

**Pull request merged**